python/CMakeLists.txt

```diff
@@ -30,19 +30,33 @@ ExternalProject_Add(
         INSTALL_COMMAND ""
 )
 
-set(BUILD_SH_IN "${CMAKE_CURRENT_SOURCE_DIR}/build_wheel.sh.in")
-set(BUILD_SH    "${CMAKE_CURRENT_BINARY_DIR}/build_wheel.sh")
-configure_file(${BUILD_SH_IN} ${BUILD_SH} @ONLY)
-
-add_custom_command(
-        OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/dist/
-        POST_BUILD
-        WORKING_DIR ${CMAKE_CURRENT_BINARY_DIR}
-        COMMAND ${CMAKE_BUILD_TOOL} -C ../ DESTDIR=python/_install install && bash build_wheel.sh
-)
+if (WIN32)
+        set(BUILD_PY_IN "${CMAKE_CURRENT_SOURCE_DIR}/build_wheel.py.in")
```

Comment:
Why do you want to use python script only for Windows? Looks like it can be used for other platforms too.

Reply:
There's an offline discussion about changes in the wheel files layout. Unification and switching to python build script for all systems could be included in that activity.